compute_endpoint/globus_compute_endpoint/endpoint/config/config.py

```diff
@@ -88,18 +88,18 @@ class Config(RepresentationMixin):
         Optional path string to the top-level directory where logs should be written to.
         Default: None
 
-    multi_tenant : bool | None
-        Designates the endpoint as a multi-tenant endpoint
+    multi_user : bool | None
+        Designates the endpoint as a multi-user endpoint
         Default: None
 
     allowed_functions : list[str] | None
         List of functions that are allowed to be run on the endpoint
 
-    force_mt_allow_same_user : bool
+    force_mu_allow_same_user : bool
         If set, override the heuristic that determines whether the uid running the
-        multi-tenant endpoint may also run user endpoints.
+        multi-user endpoint may also run user endpoints.
 
-        Normally, the multi-tenant endpoint disallows starting user endpoints with
+        Normally, the multi-user endpoint disallows starting user endpoints with
```

Comment:
In this context, changing `tenant` to `user` suggests to me that we should also modify the direct object for clarity.  Why can't a multi-user endpoint run a user endpoint?  Seems silly, on the face of it.  Suggest adding `single-` as a qualifier:

> If set, override the heuristic that determines whether the uid running the multi-user endpoint may also run single-user endpoints.
>
> Normally, the multi-user endpoint disallows starting single-user endpoints with the same UID as the parent process unless the UID has no privileges. [...]